src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -15,14 +15,14 @@
  */
 public class DeleteCommand extends Command {
 
-    public static final String COMMAND_WORD = "delete";
+    public static final String COMMAND_WORD = "deleteclient";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the client identified by the index number used in the displayed person list.\n"
```

Comment:
Should this be "in the displayed client list" instead of " displayed person list"?

Reply:
Nice spot! I've fixed it